packages/dropdowns/src/styled/items/StyledItem.ts

```diff
@@ -39,7 +39,7 @@ const getColorStyles = (props: IStyledItemProps & ThemeProps<DefaultTheme>) => {
 /**
  * 1. Allows an item to contain a positioned sub-menu.
  * 2. Reset stacking context for sub-menu css-arrows.
- **/
+ */
```

Comment:
Fine to leave this, but please don't bother with any `styled` component updates.

Reply:
Ok.